scripts/common.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+
+function getRdsStatus() {
+    for attempt in {1..10}
+    do
+        rds_status=$(aws rds describe-db-instances --db-instance-identifier ${RDS_INSTANCE} --query 'DBInstances[0].DBInstanceStatus' --output text)
+        if [[ ! -z ${rds_status} ]]; then
+            break
+        fi
+    done
+    echo ${rds_status}
+}
+
+function waitForInstanceAvailable() {
+    for attempt in {1..10}
+    do
+        current_status=$(getRdsStatus)
+        if [[ "$current_status" == "available" ]]; then
+            echo "${RDS_INSTANCE} is now available"
+            break
+        elif [[ "$current_status" != "starting" ]] && [[ "$current_status" != "rebooting" ]]; then
```

Comment:
Perhaps it would be easier to follow if the else was inverted (I think this would be equivalent):
`elif [[ "${current_status}" = "starting" ]] || [[ "${current_status}" = "rebooting" ]]; then`
`    echo  "${RDS_INSTANCE} is still starting. Waiting until available."`
`else`
`    echo "${RDS_INSTANCE} is at status $current_status. Aborting as this will not become available."`
`    break`